### DISCUSSION TITLE
Confirmation still not recieved

### DISCUSSION THREAD

**Discussion by LuukBosmann** [community/community#11225] [Copilot Conversations] [answered]
I joined the waiting list a long time ago and other people who joined later recieved access. Is there a reason as to why haven't received this access?

**Comment by M5561** [comment]
**Youre right!!!!!!!!!!!** @github fix this!!!!

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
